Source/Blazorise/Components/Table/TableHeaderCell.razor.cs

```diff
@@ -54,6 +94,11 @@ protected Task OnClickHandler( MouseEventArgs eventArgs )
     /// </summary>
     [Parameter] public int? ColumnSpan { get; set; }
 
+    /// <summary>
+    /// Fixes the header cell to the start of the table.
```

Comment:
You mention start here? Will we not have a way to fix either at the start or the end?

Reply:
🤔 In that case I would have to change the API `bool Fixed` to an enum `TableCellFixedPosition Fixed`(or similar), with

- Start
- End

values.

Reply:
So. I'm in the middle of refactoring Fixed to support Start and End but I realized it would not be an easy task. Let me explain. For the Start I just calculate offset from the left by iterating each table cell on initialize. But for the End, I cannot do that. Blazor renders cells in the order from left to right, or top to bottom, if you know what I mean. There is no such thing as reverse rendering. Any idea how to calculate the offset from the end/right position?

Reply:
Can't the end offset be calculated by the amount of existing row columns? So instead of being a fixed calc, it's a calculate as rendering happens, since columns can change, either dynamically, or like you said sequential init from blazor regular render.

Reply:
Possibly. But I don't know what performance hit that might take when there are many rows and columns.

Reply:
Could be a chance to investigate the "Defer" possibility if you have the patiente. :D

Reply:
![image](https://github.com/Megabit/Blazorise/assets/900302/2560c756-a243-4281-bd36-1bf9398396ff)

I don't think defer will help us. Maybe I didn't understand it.

Reply:
You should be able to collect everything like the comment implies.
So you have a `Table` with everything in it even before starting the rendering. At least that's my understanding, Although I haven't explored it either.
It should avoid those cases where
**Table** renders
**table row A** renders, does some calc and tells Table there's some additional re-rendering needed based on params, etc...
[...]
Or at least that's my understanding.

Reply:
Tried it. Not that easy. The problem is as before the order of rendering, from left to right. Also the bigger problem is that we really don't want to keep track of each individual row and cell in the memory. For smaller table it is not an issue but as soon as we try playing with virtualizations it becomes a problem.

I think going the JS route might be solution but I still haven't tried it. Virtualization also might be problematic.

All in all. I would rather make it like this and only have fixed cells to the left side, and don't bother with the right side. This is something that most of user would like to have anyways.

Reply:
I wouldn't be so sure, see: 
https://github.com/Megabit/Blazorise/issues/5101
User wants action buttons at the end.

Also we shouldn't forget that the naming should be `Start` and `End` instead of `Left` and `Right`. Right?

But I understand if it's complex and we don't have time yet, so better release as is then nothing at all?

Reply:
Do you want to tackle the problem yourself? Maybe I need another brain at this.

I will prepare some basic stuff from you, if yes.